File: 雨林 精简/erp/ccq/MyEclipse 8.5/zhaoyang13接口/src/com/d/Person.java
package com.d;

public class Person implements All {

	private String name;
	private int age;

	public Person(String name, int age) {
		this.name=name;
		this.age=age;
	}


	

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public int getAge() {
		return age;
	}

	public void setAge(int age) {
		this.age = age;
	}




	@Override
	public String toString() {
		return "Person [age=" + age + ", name=" + name + "]";
	}




	@Override
	public boolean ms(Object o) {
		Person person=(Person)o;
		return this.getAge()>person.getAge();
	}




	

	

}

File: SMilA/app/src/main/java/com/example/smila/Person.kt
package com.example.smila;

class Person(qq:String,name:String,profile:String=""){
    var qq:String=""
    var name:String=""
    var profile:String?=null
    init {
        this.qq=qq
        this.name=name
        this.profile=profile
    }

    override fun toString(): String {
        return "Person(qq='$qq', name='$name', profile=$profile)"
    }

}
